### PR TITLE
Add tracks & send group email

### PR DESCRIPTION
**Pull request by Ehco1996**
astitle

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -2183,6 +2183,274 @@ Status|Meaning|Description|Schema
...
+Status Code **200**
+
+Name|Type|Required|Description
+---|---|---|---|---|
+result|boolean|true|request success or fail
+msg|string|true|response message
+data|[WaiverTrack](#schemawaivertrack)|true|Waiver Request
+
+
+## Get Group Tracks
```
Comment (ll0jj0xx0) [inline source/index.html.md:2342]: ## Get Waiver Request Tracking Info

File: source/index.html.md [hunk 2]
```diff
@@ -2183,6 +2183,274 @@ Status|Meaning|Description|Schema
...
+
+r = requests.get('https://api.waiverforever.com/openapi/v2/waiverRequests/groupTrackings?group_id=oVMlLtxIZL1673829942&page=1&per_page=10', headers=headers)
+
+print(r.json())
+
+```
+
+`GET /openapi/v2/waiverRequests/groupTrackings`
+
+*GetGroupTrackings*
```
Comment (ll0jj0xx0) [inline source/index.html.md:2404]: Retrieve the status of the email requests sent for a particular waiver request.

File: source/index.html.md [hunk 3]
```diff
@@ -2183,6 +2183,274 @@ Status|Meaning|Description|Schema
 200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|Successful request|Inline
 403|[Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3)|Invalid api key|None
 
+## Send Group Emails
```
Comment (ll0jj0xx0) [inline source/index.html.md:2186]: Send Requests via Email

File: source/index.html.md [hunk 4]
```diff
@@ -2183,6 +2183,274 @@ Status|Meaning|Description|Schema
...
+
+r = requests.post('https://api.waiverforever.com/openapi/v2/waiverRequests/sendGroupEmail', json=data, headers=headers)
+
+print(r.json())
+
+```
+
+`POST /openapi/v2/waiverRequests/sendGroupEmail`
+
+*SendGroupEmail*
```
Comment (ll0jj0xx0) [inline source/index.html.md:2288]: Once a waiver request has been created, you should use this API to send email requests to your customers asking them to sign the waiver.

File: source/index.html.md [hunk 5]
```diff
@@ -2183,6 +2183,274 @@ Status|Meaning|Description|Schema
 200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|Successful request|Inline
 403|[Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3)|Invalid api key|None
 
+## Send Requests via Email
+
+> Code samples
```
Comment (Ehco1996) [inline source/index.html.md:2188]:
```suggestion
> Code samples suggestion
```

File: source/index.html.md (hunk 4 above)
Comment (Ehco1996) [reply source/index.html.md]: thanks for your review, btw, In this case, code suggestion(like this https://github.com/AriesApp/open-api-doc/pull/27#discussion_r1096644094 ) might be a better choice

Comment (ll0jj0xx0) [reply source/index.html.md]: 啥意思呀

**Review by ll0jj0xx0 — APPROVED** [review]

**Pull request merged**